### DISCUSSION TITLE
how to reinstall if I lose Nextcloud AIO password

### DISCUSSION THREAD

**Discussion by IssueFindings** [nextcloud/all-in-one#3823] [Questions] [answered]
Hello,
I lose my Nextcloud AIO password using to encrypt. I know that I can't get it back and I need to reinstall everything.
I did it under Docker (after stop and rm AIO containers I used `docker system prune` to erase all related configuration files). When I log to AIO Home page, it ask me the old Nextcloud AIO password...

So what I need to do to totally erase my Nextcloud AIO and restart everything ?

Have a nice day.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1786

**Reply by IssueFindings** [reply]
Many thanks That works very well !! Have a nice day.
